tests/plugins/imagebase/features/link.js

```diff
@@ -44,45 +44,32 @@
 		} );
 	}
 
-	function assertLinkWidget( options ) {
+	function assertLinkWidgetStatus( options, linkCount, linkCmdState, unlinkCmdState ) {
```

Comment:
I don't have today time to do full review, but it's the first thing that caught my eye: why did you expand `options` parameter to 4 parameters?

Reply:
This is just the way diff has shown it 😉 `assertLinkWidget` function is still there, but few lines below. I simply extracted common part of both functions.

Reply:
OK, but why you expanded `options` then? :P

Reply:
Actually it make sense to reuse options, true.

Reply:
Well, true, it would be a better idea to extend options for that. 🙂